OpenRA.Mods.Common/Traits/Carryall.cs

```diff
@@ -26,14 +26,14 @@ public class CarryallInfo : ITraitInfo, Requires<BodyOrientationInfo>, Requires<
 		[Desc("Delay on the ground while attaching an actor to the carryall.")]
 		public readonly int LoadingDelay = 0;
 
-		[Desc("Delay on the ground while detacting an actor to the carryall.")]
+		[Desc("Delay on the ground while detaching an actor from the carryall.")]
 		public readonly int UnloadingDelay = 0;
```

Comment:
Can we please call this `BeforeUnloadDelay` to match `Cargo`?

Reply:
Done.